src/engine/enginebuffer.cpp

```diff
@@ -179,6 +180,16 @@ EngineBuffer::EngineBuffer(QString group, ConfigObject<ConfigValue>* _config,
     connect(m_pSlipButton, SIGNAL(valueChangedFromEngine(double)),
             this, SLOT(slotControlSlip(double)),
             Qt::DirectConnection);
+    
+    m_pSlipCancelButton = new ControlPushButton(ConfigKey(m_group, "slip_cancel"));
+    m_pSlipCancelButton->setButtonMode(ControlPushButton::TOGGLE);
+    connect(m_pSlipCancelButton, SIGNAL(valueChanged(double)),
+            this, SLOT(slotControlSlipCancel(double)),
+            Qt::DirectConnection);
+    connect(m_pSlipCancelButton, SIGNAL(valueChangedFromEngine(double)),
```

Comment:
This connection is not required, since you do not change the value of m_pSlipCancelButton from the engine.